fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/ReindexResourceDAO.java

```diff
@@ -285,6 +285,16 @@ public void updateParameters(String tablePrefix, List<ExtractedParameterValue> p
         deleteFromParameterTable(connection, tablePrefix + "_latlng_values", logicalResourceId);
         deleteFromParameterTable(connection, tablePrefix + "_resource_token_refs", logicalResourceId);
         deleteFromParameterTable(connection, tablePrefix + "_quantity_values", logicalResourceId);
+        deleteFromParameterTable(connection, tablePrefix + "_profiles", logicalResourceId);
+        deleteFromParameterTable(connection, tablePrefix + "_tags", logicalResourceId);
+
+        // whole-system search params
+        deleteFromParameterTable(connection, "logical_resource_profiles", logicalResourceId);
+        deleteFromParameterTable(connection, "logical_resource_tags", logicalResourceId);
```

Comment:
note to self:  ensure that the security tables are added to both of these lists after merging mike's changes

Reply:
Updated to use a common support procedure